README.md

```diff
@@ -1 +1,29 @@
 # Valigator
+
+Valigator is a lazy validation builder library. It allows for the reuse of validation across your code in an easy-to-use manner. 
+
+Usually, when doing validation, people end up writing validation logic in the constructor which is okay but you don't really end up being able to easily share that validation functionality across classes. 
+
+The idea is to somewhat decouple validation from your classes. So you perform validation based on a set of criteria. If you fulfill the criteria you should be able to create your Model based on that data. 
```

Comment:
Perhaps drop the somewhat. It makes it sound like we are unsure if it does or not